miner/worker.go

```diff
@@ -618,39 +614,14 @@ func (w *worker) resultLoop() {
 	}
 }
 
-// timestampLoop is a loop that updates the timestamp used for blocks when it
-// is stale by more than a certain threshold.
-// TODO: Re-think this as everything comes together more.
-func (w *worker) timestampLoop() {
```

Comment:
Removing the timestamp loop here as it is now managed in the sequencer ingest loop. Instead of directly setting the chain's timestamp, we set the timestamp on the txmeta and then use that timestamp to set the time field in the block header right before block creation.